tests/console/validate_addon_extensions_repos_http_ftp.pm

```diff
@@ -0,0 +1,46 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate that http and ftp addon extensions repos added during
+# installation are enabled in the installed system.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use repo_tools 'validate_repo_properties';
+
+sub run {
```

Comment:
Everything looks great for me in this PR. The only thing that could be changed in my opinion is this test module. I guess we can split it into two test modules: validating ftp and validating http to make them more atomic and clear or what is validated where. WDYT?

Reply:
I think I didn't do it because I'm not sure how will go the atomicity with validations, because validation can be randomly different even with the same installation due to different settings left by packages or combination of virtualization + arch. However makes sense in this case, thanks for spotting it. Restarting VRs.